src/types/mod.rs

```diff
@@ -502,6 +502,115 @@ mod rewards_overview {
     }
 }
 
+pub mod block_certificates {
+    use concordium_base::hashes::BlockHash;
+
+    use super::*;
+
+    /// An aggregate signature on a `QuorumCertificate` created
+    /// by members of the finalization committee.
+    #[derive(Debug)]
+    pub struct QuorumSignature(pub [u8; 48]);
+
+    /// A quorum certificate on a block.
+    #[derive(Debug)]
+    pub struct QuorumCertificate {
+        /// `Hash` of the block that this `QuorumCertificate`
+        /// certifies.
+        pub block_hash:          BlockHash,
+        /// `Round` of the block that this `QuorumCertificate`
```

Comment:
```suggestion
        /// [`Round`] of the block that this `QuorumCertificate`
```

if it is imported, otherwise link to it, e.g., 
```
[`Round`](concordium_base::types::Round)
```
or wherever it is.